spec/settings.yml

```diff
@@ -26,3 +26,12 @@ array:
     name: first
   -
     name: second
+
+defaults: &defaults
+  base: base definition
+  other: that other thing
+
+overriden:
+  <<: *defaults
+  base: overrriden definition
```

Comment:
Did you mean to add explicit tests for this, or just that it loads without error?

Reply:
I was just making sure it loaded w/out error, but I can add a quick confirmation test.